.circleci/config.yml

```diff
@@ -408,6 +408,8 @@ jobs:
       docker_layer_caching: true # we rely on this for faster builds, and actively warm it up for builds with common stages
     steps:
       - checkout
+      - run:
+          command: git submodule update --init
```

Comment:
Looks like `install-contracts-dependencies` also does it, can it be reused?

Reply:
Yes, the previous is simple but the current is more structured, and they both pass the [test](https://app.circleci.com/pipelines/circleci/VJoJbnHRXJiFR13mWFvRfZ/SHyYDZiLFYMtveJSZVFKbw/75/workflows/8c45fb9a-d059-4580-94b5-b9869f515abe/jobs/1435). @qzhodl and @qizhou?

Reply:
If I understand correctly, @blockchaindevsh was referring to not reusing the `git submodule update --init` command when mentioning “previous”

Reply:
Yeah, not reusing results in fewer lines of change here, will be easier to merge in future.

Reply:
I agree, let's keep it simple